util/app-config/src/cli.rs

```diff
@@ -350,6 +355,33 @@ fn init() -> App<'static, 'static> {
         )
 }
 
+fn peer_id() -> App<'static, 'static> {
+    SubCommand::with_name(CMD_PEERID)
+        .about("about peer id, base on Secp256k1")
```

Comment:
When I try `ckb --help`, I found that all other about messages were capitalized.